lib/playerlist.js

```diff
@@ -59,39 +59,30 @@ class PlayerList extends LitElement {
   init (bot) {
     const playerList = this.shadowRoot.querySelector('#playerlist-container')
 
-    Object.values(bot.players).forEach(player => {
-      addPlayer(player)
-    })
+    Object.values(bot.players).forEach(addPlayer)
 
     this.isOpen = false
 
-    const self = this
-
-    document.onkeydown = function (e) {
-      e = e || window.event
+    document.onkeydown = e => {
+      e ??= window.event
       if (e.key === 'Tab') {
         showList(true)
         e.preventDefault()
       }
     }
 
-    document.onkeyup = function (e) {
-      if (!self.isOpen) return
-      e = e || window.event
+    document.onkeyup = e => {
+      if (!this.isOpen) return
+      e ??= window.event
       if (e.key === 'Tab') {
         showList(false)
         e.preventDefault()
       }
     }
 
     function showList (shouldShow = true) {
-      if (shouldShow) {
-        playerList.style.display = 'block'
-      } else {
-        playerList.style.display = 'none'
-      }
-
-      self.isOpen = shouldShow
+      playerList.style.display = shouldShow ? 'block' : 'none'
+      this.isOpen = shouldShow
```

Comment:
Cannot set property 'isOpen' of undefined (this is line 85)

Reply:
this needs fixing

Reply:
I think it's fixed now